lib/api_callable.js

```diff
@@ -190,19 +190,32 @@ function retryable(aFunc, retry, otherArgs) {
   return function retryingFunc(argument, callback) {
     var canceller;
     var timeoutId;
-    var error = new Error('Retry total timeout exceeded before any' +
-        'response was received');
+    var error = new Error();
```

Comment:
Ooh, I've forgotten about that...

Reconsidering about that, looks like line 230-231 are redundant -- the `note` says the same thing as in the `message` field. It was added previously for a different reason, and probably I rewrote to the current form without considering deeply. Please remove them.